src/engine/sidechain/enginebroadcast.cpp

```diff
@@ -594,21 +621,20 @@ bool EngineBroadcast::writeSingle(const unsigned char* data, size_t len) {
     // We are already synced by EngineNetworkstream
     setFunctionCode(8);
     int ret = shout_send_raw(m_pShout, data, len);
-    if (ret < SHOUTERR_SUCCESS && ret != SHOUTERR_BUSY) {
-        // in case of bussy, frames are queued and queue is checked below
-        qDebug() << "EngineBroadcast::write() header error:"
-                 << ret << shout_get_error(m_pShout);
+    if (ret == SHOUTERR_BUSY) {
+        // in case of busy, frames are queued
+        // try to flush queue after a short sleep
+        qDebug() << "EngineBroadcast::writeSingle() SHOUTERR_BUSY, trying again";
+        QThread::msleep(10);
```

Comment:
The server has reported to be bussy. 
This is just a guess for the time until the server is back from busy. I have no valid test data for this but it seams be an improvement.

I have left a comment.